browser/browser.go

```diff
@@ -2,8 +2,24 @@ package browser
 
 import (
 	"context"
+	"regexp"
 )
 
 type Browser interface {
-	Browse(cxt context.Context) chan *LocalAssetFile
+	Browse(cxt context.Context, excludeFiles []string) chan *LocalAssetFile
```

Comment:
Your implementation breaks the unit tests.

I would prefer changing the NewLocalFiles function  rather changing the browser interface.